### ISSUE TITLE
Include Files references are only lower cased

### ISSUE THREAD

**Issue by ChrSteinert** [fsprojects/Paket#1676]
### Description

Everything in the include files is lowercase, except the name of the file itself (yet, the path is).

### Repro steps

1. Set up a "clean" Paket directory (following the steps from the getting started page).
2. Use the following paket.dependencies file:
```
source https://nuget.org/api/v2

nuget EntityFramework
```
3. Generate include scripts via the command `paket generate-include-scripts framework net46`.

### Expected behavior

In the `paket-files\include-scripts\net46` folder is a new file `include.EntityFramework.fsx` (or `*.csx`) generated with the following content:
```
#r "System.ComponentModel.DataAnnotations" 
#r "../../../packages/EntityFramework/lib/net45/EntityFramework.dll" 
#r "../../../packages/EntityFramework/lib/net45/EntityFramework.SqlServer.dll" 
printfn "%s" "Loaded EntityFramework"
```

### Actual behavior

In the `paket-files\include-scripts\net46` folder is a new file `include.EntityFramework.fsx` (or `*.csx`) generated with the following content:
```
#r "System.ComponentModel.DataAnnotations" 
#r "../../../packages/entityframework/lib/net45/EntityFramework.dll" 
#r "../../../packages/entityframework/lib/net45/EntityFramework.SqlServer.dll" 
printfn "%s" "Loaded entityframework"
```
(folder and package names are lowercased)
### Related information 

* Running Pre-release 3.0.0-beta036

**Comment by smoothdeveloper** [comment]
@ChrSteinert going to look at that, I suspect there is a confusion between the package identifier for comparison (which is lowercased in paket codebase) and the actual package folder name.

Thanks for first bug report on this feature :+1:

**Comment by smoothdeveloper** [comment]
fix in #1678

**Comment by ChrSteinert** [comment]
Leave a comment
